### PR TITLE
add stars

### PR DESCRIPTION
**Pull request by SerhiyPaziyc**
- [DEMO LINK](https://SerhiyPaziyc.github.io/layout_stars/)
- [TEST REPORT LINK](https://SerhiyPaziyc.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,40 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```
Comment (anastasiiavorobiova) [inline src/style.css:22]:
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,40 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (anastasiiavorobiova) [inline src/style.css:40]:
```suggestion

```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Comment by SerhiyPaziyc** [comment]
- [DEMO LINK](https://SerhiyPaziyc.github.io/layout_stars/)
- [TEST REPORT LINK](https://SerhiyPaziyc.github.io/layout_stars/report/html_report/)

**Comment by SerhiyPaziyc** [comment]
redid

**Comment by SerhiyPaziyc** [comment]
redid

**Pull request closed without merging**